web/ui/templates/_base.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name=”robots” content=”noindex,nofollow”>
```

Comment:
Thanks! Could you use the regular `"` quotes here instead of `”`?

Reply:
Amended. Not sure how you want to operate in this repo, should I rebase?